src/Legacy/Storage.php

```diff
@@ -1831,12 +1817,11 @@ protected function executeGetContentQueries($decoded)
      *
      * @param string $textquery
      * @param string $parameters
-     * @param array  $pager
      * @param array  $whereparameters
      *
      * @return array
      */
-    public function getContent($textquery, $parameters = '', &$pager = [], $whereparameters = [])
+    public function getContent($textquery, $parameters = '', $whereparameters = [])
     {
```

Comment:
I'm not sure what do you suggest. Because it is a reference we'll have declared but unused variables on calls.